website/docs/tutorials/02-installation.md

```diff
@@ -4,45 +4,50 @@ sidebar_label: Installation
 
 # Installation
 
+Protostar is a valuable tool for developing StarkNet smart contracts, and it is easy to install on Linux and Mac systems. In this section, we will walk through the process of installing and upgrading Protostar, as well as the requirements for using the tool.
+
 ## Requirements
-Protostar requires a [git](https://git-scm.com/) executable to be specified in the `$PATH`.
+To use Protostar's dependency management commands, you must have git installed and added to your `$PATH` environment variable.
+
+The `$PATH` is an environment variable that specifies a list of directories in which the system looks for executables.
+If you do not have git in your `$PATH`, you will need to add the directory where git is installed in order to use Protostar.
 
 ## Linux and Mac
-1. Copy and run in a terminal the following command:
+1. Open a terminal and enter the following command:
 ```console
 curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/install.sh | bash
 ```
-2. Restart the terminal.
-3. Run `protostar -v` to check Protostar and [cairo-lang](https://pypi.org/project/cairo-lang/) version.
+2. Close and reopen the terminal.
+3. To check the version of Protostar and Cairo that you have installed, run `protostar -v`.
 
 ### Specifying version
 
-If you want to specify a version of protostar to install run the following command with the requested version:
+If you want to install a specific version of Protostar, run the following command with the desired version number:
 
 ```console
 curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/install.sh | bash -s -- -v 0.3.2
 ```
 
-## Windows
-Not supported. 
+## Windows Compatibility
+Protostar is not currently supported on Windows.
 
 # Upgrading Protostar
 To upgrade Protostar, run:
 ```shell
-$ protostar upgrade
+protostar upgrade
 ```
 
 # Protostar releases
 
 Protostar follows a biweekly release schedule.
 Each release may include new features, enhancements, bug fixes, deprecations and breaking changes.
-Consult [release notes](https://github.com/software-mansion/protostar/releases) for detailed information about each release.
-Protostar prints a link to new release before upgrading.
+For detailed information about each release, consult the [release notes](https://github.com/software-mansion/protostar/releases).
+A link to the release notes for the latest version of Protostar will be displayed when a new release is available.
```

Comment:
Protostar will let you know it new version is available

Reply:
I prefer the version in the PR because of the relation with the previous sentence. Also, "letting sb know" could be interpreted as system notifications.

Reply:
>be interpreted as system notifications.

Reply:
> be interpreted as system notifications.

I don't think so, I just don't think we need this level of detail, it is hard to read.

Reply:
[shorten the sentence about a new release](https://github.com/software-mansion/protostar/pull/1306/commits/0d9731dfd917a53a84fb9b7c4fe50a5cd95a7a44)